### PR TITLE
Add mock actions script

### PR DESCRIPTION
**Pull request by 0xGabi**

### CODE REVIEW

File: packages/hardhat/abi/@aragon/apps-finance/contracts/Finance.sol/Finance.json [hunk 1]
```diff
@@ -0,0 +1,1009 @@
+[
```
Comment (rperez89) [inline packages/hardhat/abi/@aragon/apps-finance/contracts/Finance.sol/Finance.json:1]: why we need apps abis that are not going to be installed on the gardens template?

**Review by rperez89 — COMMENTED** [review]
@0xGabi i am noticing a bunch of abis that not sure if we need, could we also consider split this pr into 2 different ones so we can review it in a better way? something like one with the abis and other with the really mock action scripts functionality

File: packages/hardhat/abi/@aragon/apps-finance/contracts/Finance.sol/Finance.json (hunk 1 above)
Comment (0xGabi) [reply packages/hardhat/abi/@aragon/apps-finance/contracts/Finance.sol/Finance.json]: Yes, you were right, there was a lot of ABIs not needed. I revert that change on #81

**Comment by 0xGabi** [comment]
I'll close this one for now.

**Pull request closed without merging**